store/bolt/store.go

```diff
@@ -0,0 +1,75 @@
+package bolt
+
+import (
+	"encoding/binary"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/boltdb/bolt"
+)
+
+//go:generate protoc --gogo_out=. internal/internal.proto
```

Comment:
This line is for the `go generate` command. If you call `go generate ./...` from the command line then the `go` tool chain will automatically find all the `go:generate` lines and run them. It's really useful for generating Go code.